homeassistant/components/vesync/strings.json

```diff
@@ -56,6 +56,11 @@
         "name": "Mist level"
       }
     },
+    "switch": {
```

Comment:
I think `display` would be a better term than `screen`, it also matches the setter api.